vagrant/provisioning/roles/pentaho-configuration/tasks/main.yml

```diff
@@ -286,6 +286,21 @@
     delay: 5
     timeout: 300
 
```

Comment:
@martinbozinovski I'm guessing the playbook will never reach this point as it will probably fail in the previous task. We should put a handler in the task "wait for pentaho webapp to really be available" if it fails not to fail the whole playbook but continue on the next task that you have created.